scripts/runtests.sh

```diff
@@ -254,6 +254,28 @@ python3 scripts/check-yaml-fields.py "$tmpdir"/var/log/installer/autoinstall-use
         'autoinstall.source.id="ubuntu-server-minimal"'
 grep -q 'finish: subiquity/Install/install/postinstall/run_unattended_upgrades: SUCCESS: downloading and installing security updates' $tmpdir/subiquity-server-debug.log
 
+clean
+LANG=C.UTF-8 timeout --foreground 60 \
+    python3 -m subiquity.cmd.tui \
+    --dry-run \
+    --output-base "$tmpdir" \
+    --machine-config examples/simple.json \
+    --autoinstall examples/autoinstall-ad.yaml \
+    --kernel-cmdline autoinstall \
+    --source-catalog examples/mixed-sources.yaml &
+sub_pid=$!
+until [[ -S "$tmpdir/socket" || $(ps -p "$sub_pid" > /dev/null ) == 1 ]]
```

Comment:
This construction with ps does not correctly screen for the pid disappearing.  you're always comparing an empty string to 1.